website/docs/components/side-nav/partials/code/component-api.md

```diff
@@ -14,6 +14,30 @@ This is the full-fledged component (responsive and animated).
   <C.Property @name="<:footer>" @type="named block">
     A named block where the content for the “footer” section of the SideNav is rendered. It yields the value of `isMinimized` too.
   </C.Property>
+  <C.Property @name="isResponsive" @type="boolean" @default="true">
+    Controls whether the SideNav is responsive to viewport changes. It can be programmatically turned off passing `false`.
+    <br>
+    <em>Notice: even if the @isResponsive parameter is set to false, some JavaScript is executed anyway in the background, and events listeners are attached to some DOM elements (even if these functionality are not used).</em>
+  </C.Property>
+  <C.Property @name="hasA11yRefocus" @type="boolean" @default="true">
+    Controls whether a "navigator narrator" and a "skip link" are added to the navigation (provided by the [`ember-a11y-refocus` Ember addon](https://github.com/ember-a11y/ember-a11y-refocus)). It can be programmatically turned off passing `false`.
```

Comment:
```suggestion
    Controls whether a "navigator narrator" and a "skip link" are added to the navigation (provided by the [`ember-a11y-refocus` Ember addon](https://github.com/ember-a11y/ember-a11y-refocus)). It can be programmatically turned off by passing `false`.
```